Cargo.toml

```
@@ -9,7 +9,7 @@ readme = "README.md"
 homepage = "https://github.com/snapview/tungstenite-rs"
 documentation = "https://docs.rs/tungstenite/0.21.0"
```

```suggestion
documentation = "https://docs.rs/tungstenite/0.22.0"
```

If the version is raised, we must also update the documentation link.